_traefik2_labels.yml.jinja

```diff
@@ -154,12 +154,13 @@
 
       {#- Longpolling router #}
       {%- if not domain_group.path_prefixes %}
+      {%- set longpolling_route = "/longpoling/" if odoo_version < 16 else "/websocket" -%}
       {{-
         router(
           domain_group=domain_group,
           key=key,
           suffix="longpolling",
-          rule="%s && PathPrefix(`/longpolling/`)" % domains_rule(domain_group),
+          rule="%s && PathPrefix(`%s`)" % (domains_rule(domain_group), longpolling_route),
```

Comment:
According to my tests, the rule for <16 should be ```PathPrefix(`/longpolling/`)``` as it was, but the rule for >= 16 should be ```Path(`/websocket`)```.

Reply:
Line 157 should take care of that, but apparently I mistyped `/longpolling/` as `/longpoling/`, should be fixed now.

Reply:
I mean that you don't want ```PathPrefix(`/websocket`)```, but ```Path(`/websocket`)``` instead.

Reply:
I misread your first comment, sorry about that.  `Path` does indeed seem the correct rule, changed it.